Out-of-schedule Executive vote - July 4, 2024.md (#2685)
makerdao/community: governance/votes/Executive vote - July 4, 2024.md

---

> title: Template - [Executive Vote] Out-of-Schedule Executive Vote - Approve LITE-PSM-USDC DAO Resolution - July 04, 2024

Reviewer: ```suggestion
title: Template - [Executive Vote] Out-of-Schedule Executive Vote - Approve LitePSM (LITE-PSM-USDC-A) DAO Resolution - July 04, 2024
```

> summary: Approving a DAO Resolution pertaining to the existing Coinbase Custody arrangement (RWA014-A) and the new Lightweight Peg Stability Moodule (LITE-PSM-A). 

Reviewer 2: I would specify what the DAO Resolution is defining. "Pertaining" only indicates that it refers to an existing arrangement and the LitePSM, but doesn't indicated in what terms. I'd suggest to put: "Approval of DAO Resolution defining the migration from the existing Coinbase Custody Agreement (PSM-USDC-A/ RWA014-A) to the new Lightweight Peg Stability Module (LITE-PSM-USDC-A/ LitePSM)."

> summary: Approving a DAO Resolution pertaining to the existing Coinbase Custody arrangement (RWA014-A) and the new Lightweight Peg Stability Moodule (LITE-PSM-A). 

Reviewer: ```suggestion
summary: Approve DAO Resolution authorizing the actions needed for the setup of Coinbase Prime Web3 Wallet (W3W) and migration from PSM-USDC-A to the new Lightweight Peg Stability Module (LITE-PSM-USDC-A).
```

date: 2024-07-04T00:00:00.000Z
address: "$spell_address"

---

> # [Executive Proposal] Out-of-Schedule Executive Vote - Approve LITE-PSM-USDC DAO Resolution - July 04, 2024

Reviewer: ```suggestion
# [Executive Proposal] Out-of-Schedule Executive Vote - Approve LitePSM (LITE-PSM-USDC-A) DAO Resolution - July 04, 2024
```

> The Governance Facilitators and Dewiz have placed an executive proposal into the voting system. MKR Holders should vote for this proposal if they support the following alterations to the Maker Protocol.

Reviewer: ```suggestion
The Governance Facilitators, Dewiz, and Sidestream have placed an executive proposal into the voting system. MKR Holders should vote for this proposal if they support the following alterations to the Maker Protocol.
```

We also have to add the reviewer here (Sidestream)

If you are new to voting in the Maker Protocol, please see the [voting guide](https://manual.makerdao.com/governance/voting-in-makerdao/on-chain-governance) to learn how voting works.

---

## Executive Summary

> If this executive proposal passes, the following actions will occur within the Maker Protocol:

Reviewer 2: As per the template "actions" should be in bold.

> If this executive proposal passes, the following actions will occur within the Maker Protocol:

Reviewer: ```suggestion
If this executive proposal passes, the following **actions** will occur within the Maker Protocol:
```

> - The "Approve LITE-PSM-USDC" DAO Resolution will be approved.

Reviewer 2: I think we are redundant repeating the "approval" action. I'd leave it as: "LITE-PSM-USDC DAO Resolution will be approved, triggering the actions listed below". I would then list the actions/ scope of the Resolution in Proposal Details (see my comment below).

> 

Reviewer 2: As per the template, we're missing the statement:

**Voting for this executive proposal will place your MKR in support of the actions outlined above.**

> Unless otherwise noted, the changes and additions listed above are subject to the [GSM Pause Delay](https://manual.makerdao.com/parameter-index/core/param-gsm-pause-delay). This means that if this executive proposal passes, the changes and additions listed above will only become active in the Maker Protocol after the GSM Pause Delay has expired. The GSM Pause Delay is currently set to **30 hours**.

Reviewer: ```suggestion
Voting for this executive proposal will place your MKR in support of the changes outlined above.

Unless otherwise noted, the changes and additions listed above are subject to the [GSM Pause Delay](https://manual.makerdao.com/parameter-index/core/param-gsm-pause-delay). This means that if this executive proposal passes, the changes and additions listed above will only become active in the Maker Protocol after the GSM Pause Delay has expired. The GSM Pause Delay is currently set to **30 hours**.
```

If this executive proposal does not pass within 30 days, then it will expire and can no longer have any effect on the Maker Protocol.

---

## Proposal Details

> ### Approve "Approve LITE-PSM-USDC" DAO Resolution

Reviewer 2: Same comment as above - it sounds weird and redundant that we are approving an approval. I'd leave it at "Approve LITE-PSM-USDC DAO Resolution"

> ### Approve "Approve LITE-PSM-USDC" DAO Resolution

Reviewer: ```suggestion
### Approve LitePSM (LITE-PSM-USDC-A) DAO Resolution
```

> - **Authorization:** [Forum post]()

Reviewer: ```suggestion
- **Authorization:** [Forum post](http://forum.makerdao.com/t/coinbase-web3-wallet-legal-overview/24577/2)
```

- **Proposal:** [Forum post](https://forum.makerdao.com/t/coinbase-web3-wallet-legal-overview/24577)

> If this executive proposal passes, the following DAO Resolution pertaining to the approval of LITE-PSM-USDC in the context of RWA014-A ([Coinbase Custody arrangement](https://forum.makerdao.com/t/coinbase-custody-legal-assessment/20384)) and the transition from the current custodial solution to a new non-custodial solution that leverages the Coinbase Web3 Wallet (the “W3W”) and the Lightweight Peg Stability Module ([LitePSM](https://forum.makerdao.com/t/litepsm-lite-psm-usdc-a-introduction-and-overview/24512)) will be approved:

Reviewer 2: I would make the elements of DAO Resolution very clear here. My proposal would be to formulate the Proposal Details as follows:
"If this executive proposal passes, LITE-PSM-USDC DAO Resolution will be approved, triggering the following actions:
- [Legal Agreements](https://forum.makerdao.com/t/coinbase-web3-wallet-legal-overview/24577) will enter into effect.
- Lightweight Peg Stability Module (LITE-PSM-USDC-A/ LitePSM) will be implemented with defined [features and parameters](https://forum.makerdao.com/t/litepsm-lite-psm-usdc-a-introduction-and-overview/24512)
- Migration from PSM-USDC-A to LITE-PSM-USDC-A will be triggered, following the defined [migration and offboarding plan](https://forum.makerdao.com/t/litepsm-lite-psm-usdc-a-introduction-and-overview/24512)

> If this executive proposal passes, the following DAO Resolution pertaining to the approval of LITE-PSM-USDC in the context of RWA014-A ([Coinbase Custody arrangement](https://forum.makerdao.com/t/coinbase-custody-legal-assessment/20384)) and the transition from the current custodial solution to a new non-custodial solution that leverages the Coinbase Web3 Wallet (the “W3W”) and the Lightweight Peg Stability Module ([LitePSM](https://forum.makerdao.com/t/litepsm-lite-psm-usdc-a-introduction-and-overview/24512)) will be approved:

Reviewer: ```suggestion
If this executive proposal passes, the following DAO Resolution authorizing the actions needed for the setup of Coinbase Prime Web3 Wallet (W3W) and migration from PSM-USDC-A to the new [Lightweight Peg Stability Module (LITE-PSM-USDC-A)](https://forum.makerdao.com/t/litepsm-lite-psm-usdc-a-introduction-and-overview/24512)) will be approved:
```

> - Approve LITE-PSM-USDC Dao Resolution with IPFS hash [QmX2CnZcsZJtgJUdkpwsAd1bXEaFuxFUaXkqgDkZa79idA](https://ipfs.io/ipfs/QmX2CnZcsZJtgJUdkpwsAd1bXEaFuxFUaXkqgDkZa79idA).

Reviewer 2: I think it might be slightly confusing for the voters, so I'd make the Proposal Details more specific and formulate them as follows:

If this executive proposal passes, [LITE-PSM-USDC DAO Resolution](https://ipfs.io/ipfs/QmX2CnZcsZJtgJUdkpwsAd1bXEaFuxFUaXkqgDkZa79idA) will be approved, authorizing the following actions required to deploy the Coinbase Prime Web3 Wallet (W3W):

Execution of [Legal Agreements](https://forum.makerdao.com/t/coinbase-web3-wallet-legal-overview/24577).
Implementation of [Lightweight Peg Stability Module (LITE-PSM-USDC-A/ LitePSM)](https://forum.makerdao.com/t/litepsm-lite-psm-usdc-a-introduction-and-overview/24512) with any related required setup.
Migration from PSM-USDC-A to LITE-PSM-USDC-A, following the defined [migration and offboarding plan](https://forum.makerdao.com/t/litepsm-lite-psm-usdc-a-introduction-and-overview/24512).

> - Approve LITE-PSM-USDC Dao Resolution with IPFS hash [QmX2CnZcsZJtgJUdkpwsAd1bXEaFuxFUaXkqgDkZa79idA](https://ipfs.io/ipfs/QmX2CnZcsZJtgJUdkpwsAd1bXEaFuxFUaXkqgDkZa79idA).

Reviewer: ```suggestion
- Approve LITE-PSM-USDC-A Dao Resolution with IPFS hash [QmX2CnZcsZJtgJUdkpwsAd1bXEaFuxFUaXkqgDkZa79idA](https://ipfs.io/ipfs/QmX2CnZcsZJtgJUdkpwsAd1bXEaFuxFUaXkqgDkZa79idA).
```
